From FEDORA_PATCHES Mon Sep 17 00:00:00 2001
From: Elena Zannoni <ezannoni@redhat.com>
Date: Fri, 27 Oct 2017 21:07:50 +0200
Subject: gdb-6.3-test-self-20050110.patch

;; Get selftest working with sep-debug-info
;;=fedoratest

2004-02-23  Elena Zannoni  <ezannoni@redhat.com>

        * gdb.gdb/selftest.exp: Make sure that the debug directory is
        set up properly.
        * gdb.gdb/complaints.exp: Ditto.
        * gdb.gdb/xfullpath.exp: Ditto.
        * gdb.gdb/observer.exp: Ditto.

diff --git a/gdb/testsuite/lib/selftest-support.exp b/gdb/testsuite/lib/selftest-support.exp
--- a/gdb/testsuite/lib/selftest-support.exp
+++ b/gdb/testsuite/lib/selftest-support.exp
@@ -151,18 +151,18 @@ proc do_self_tests {function body} {
     }
 
     # Remove any old copy lying around.
-    remote_file host delete $xgdb
+    #remote_file host delete $xgdb
 
     gdb_start
-    set file [remote_download host $GDB_FULLPATH $xgdb]
+    #set file [remote_download host $GDB_FULLPATH $xgdb]
 
-    set result [selftest_setup $file $function]
+    set result [selftest_setup $GDB_FULLPATH $function]
     if {$result == 0} then {
 	set result [uplevel $body]
     }
 
     gdb_exit
-    catch "remote_file host delete $file"
+    #catch "remote_file host delete $file"
 
     if {$result < 0} then {
 	warning "Couldn't test self"
